### ISSUE TITLE
Mmmmmmm target dummy / Mekanism Mekasuit incompatible

### ISSUE THREAD

**Issue by jnaset** [NillerMedDild/Enigmatica6#3456]
### Modpack Version

0.5.11

### Describe your issue.

A (Mmmmmmmmmm) target dummy can be dressed in a Mekanism Mekasuit, but the suit becomes invisible, provides no damage reduction to the dummy, and does not lose any energy when taking hits.

Ideally the Mekasuit would be visible on the dummy like other armor is, and provide the same damage prevention to the dummy that it does to a player.

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Server

**Comment by BooBeerb** [comment]
I don't believe this is something the team can do anything about, you would best be off reporting this directly to the devs of Mmm(many m's)

**Comment by BooBeerb** [comment]
Oh my bad, I just noticed the version. M(many m's) has updated a few times between 0.5.11 and 0.5.16, I believe

**Comment by NillerMedDild** [comment]
I reported it here: https://www.curseforge.com/minecraft/mc-mods/mmmmmmmmmmmm/issues/7
